_develop/connect/get-started.md

```diff
@@ -142,4 +142,24 @@ Then, you can use API methods under the `blockstackConnect` global variable:
 ```javascript
 const authOptions = { /** See docs above for options */ };
 blockstackConnect.showBlockstackConnect(authOptions);
+```
+
+## Handling redirect fallbacks
+
+Connect is built to use popups with the `window.postMessage` API, which provides a much better and seamless user experience. However, there are times when this flow can fail. For example, the popup may be blocked, or the `window.postMessage` API might not work properly (which often happens on mobile browsers).
```

Comment:
nit, might link to https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage